src/cypress/integration/Form/TextArea/textarea-error.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { textareaCopy } from 'shared/data/copyContent';
+
+describe('TextArea Error tests', () => {
+  before(() => {
+    cy.visitStorybook();
+    cy.loadStory('components-form-textarea', 'error');
+  });
+
+  it('Should be displayed', () => {
+    cy.findByText(textareaCopy.pirateLore);
+  });
+
+  it('Should have an error message', () => {
+    cy.findByText(textareaCopy.errorMessage);
+  });
+
+  it('Should remove the error message once a value is entered', () => {
+    cy.findByText(textareaCopy.pirateLore)
+      .siblings('textarea')
+      .type('I feel a mean scurvy coming on...');
```

Comment:
This should be in the `textareaCopy` too so we aren't relying on a hard-coded string here.

Reply:
Would setting it to a variable work? Like: 
`it('Should remove the error message once a value is entered', () => {
    const pirateSpeak = 'I feel a mean scurvy coming on...';
    cy.findByText(textareaCopy.pirateLore)
      .siblings('textarea')
      .type(pirateSpeak);`

Because otherwise that string isn't going to shared between the test file and the stories file, just the testing suite. I can put it in the copy if you prefer that way though.

Reply:
Ah, my bad. If this is the case, let's just make it a scoped variable to this test block 👍